src/gentropy/datasource/finngen_ukb_meta/study_index.py

```diff
@@ -15,12 +19,14 @@ def from_source(
         cls: type[FinngenUkbMetaStudyIndex],
         spark: SparkSession,
         raw_study_index_path_from_tsv: str,
+        efo_curation_mapping_url: str = FinngenStudiesConfig().efo_curation_mapping_url,
     ) -> StudyIndex:
         """This function ingests study level metadata from FinnGen UKB meta-analysis.
 
         Args:
             spark (SparkSession): Spark session object.
             raw_study_index_path_from_tsv (str): Raw study index path.
+            efo_curation_mapping_url (str): URL to the EFO curation mapping file.
```

Comment:
I assume the EFOs map to the FINNGEN_R11 mapping?

Reply:
Yes, that's right, and I have verified this works as expected